omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -183,13 +183,22 @@
     retries 10
   end
 
-  execute "#{rmq_plugins} enable rabbitmq_management" do
-    environment (rabbitmq_env)
-    user opc_username
-    not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
-    # management plugin needs a rabbit restart
-    notifies :restart, 'runit_service[rabbitmq]', :delayed
-    retries 10
+  if rabbitmq['management_enabled']
+    execute "#{rmq_plugins} enable rabbitmq_management" do
+      environment (rabbitmq_env)
+      user opc_username
+      not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
+      # management plugin needs a rabbit restart
+      notifies :restart, 'runit_service[rabbitmq]', :delayed
+      retries 10
+    end
+  else
+    execute "#{rmq_plugins} disable rabbitmq_management" do
```

Comment:
Should we additionally have a `not_if` block here on this service? What happens if we run `disable rabbitmq_management` multiple times w/o it enabled? We'll also restart rabbit on every chef-client run, which isn't ideal.